site/pom.xml

```diff
@@ -12,6 +12,10 @@
 		<version>0.8.0-SNAPSHOT</version>
 	</parent>
 
+	<properties>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
```

Comment:
none of this is under jbosstools so i can't push. and the jobname could be changed without this fix afaics.

but jiri merged so also should be fine to go forward.